Makefile

```
@@ -100,6 +100,13 @@ bundle:
 	sed -i -e "s/CACHE_VERSION =.*/CACHE_VERSION = $$(awk '/CACHE_VERSION =/ { print 1+$$4 }' lnbits/static/js/service-worker.js)/" \
 		lnbits/static/js/service-worker.js
 
+checkbundle:
+	cp lnbits/static/bundle.min.js lnbits/static/bundle.min.js.old
+	cp lnbits/static/bundle.min.css lnbits/static/bundle.min.css.old
+	make bundle
+	diff -q lnbits/static/bundle.min.js lnbits/static/bundle.min.js.old || exit 1
+	diff -q lnbits/static/bundle.min.css lnbits/static/bundle.min.css.old || exit 1
+
```

nitpick: remove temp files if no error
```suggestion
rm lnbits/static/bundle.min.js.old
rm lnbits/static/bundle.min.css.old
```